controllers/remote/fake/cluster.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fake
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// NewFakeClusterClient returns the same client passed as input, as output. It is assumed that the client is a
```

Comment:
```suggestion
// NewClusterClient returns the same client passed as input, as output. It is assumed that the client is a
```

Reply:
Following c-r here, do you prefer this instead?

Reply:
Yes, because the package's name is fake, so when you use it (unaliased), it's `fake.NewClusterClient` instead of `fake.NewFakeClusterClient`.